.github/workflows/release.yml

```
@@ -0,0 +1,61 @@
+name: Weekly Release
+
+on:
+  workflow_dispatch:
+  schedule:
+    # At 00:00 on Monday
+    - cron: "0 0 * * 1"
+
+jobs:
+  bump-version-and-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # Fetch all history for all branches and tags
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: "14"
+
+      - name: Install Dependencies
+        run: npm install
+
+      - name: Create Tag
+        id: tag_version
+        run: |
+          # Extracts the latest version tag, bumps the minor version
+          LATEST_VERSION=$(git describe --tags `git rev-list --tags --max-count=1`)
+          MAJOR=$(echo $LATEST_VERSION | cut -d. -f1)
+          MINOR=$(echo $LATEST_VERSION | cut -d. -f2)
```

We've once used a tag that doesn't follow this format: `v-colocation-test`.
We could add a guard that checks for the expected format, so that the action fails if the format is weird and we don't get a release with a weird name. For example:
```suggestion
          LATEST_VERSION=$(git describe --tags `git rev-list --tags --max-count=1`)
          if ! [[ "$LATEST_VERSION" =~ ^v[0-9]+\.[0-9]+\..* ]]; then
            echo "Invalid tag format, cannot bump version of: $LATEST_VERSION"
            exit 1
          fi
          MAJOR=$(echo $LATEST_VERSION | cut -d. -f1)
          MINOR=$(echo $LATEST_VERSION | cut -d. -f2)
```